src/routes/api/users.js

```diff
@@ -1,10 +1,11 @@
+
 const mongoose = require("mongoose");
 const router = require("express").Router();
 const passport = require("passport");
 const User = mongoose.model("User");
 
-router.get("/user", function(req, res, next) {
-    User.findById(req.payload.id)
+router.get("/user/:id", function(req, res, next) {
+    User.findById(req.params.id)
         .then(function(user) {
             if (!user) {
                 return res.sendStatus(401);
```

Comment:
I think the status code should be 404 because we are testing for not found.
401 is unauthorized.